Casks/b/bilibili.rb

```diff
@@ -1,17 +1,17 @@
 cask "bilibili" do
-  version "1.13.5.3377"
-  sha256 :no_check
+  version "1.13.5"
+  sha256 "c3f1f43bfc8836cb27d8acbff24834129366a4e30920c3113cccb5f9e70509ac"
 
-  url "https://dl.hdslb.com/mobile/fixed/pc_electron_mac/bili_mac.dmg",
+  url "https://dl.hdslb.com/mobile/fixed/pc_electron_mac/bili_mac.dmg?v=#{version}",
       verified: "dl.hdslb.com/"
   name "Bilibili"
   name "哔哩哔哩官方客户端"
   desc "Official bilibili video streaming and sharing platform"
   homepage "https://app.bilibili.com/"
 
   livecheck do
-    url :url
-    strategy :extract_plist
+    url "http://api.bilibili.com/x/elec-frontend/update/latest.yml"
```

Comment:
Let's use the secure connection if available
```suggestion
    url "https://api.bilibili.com/x/elec-frontend/update/latest.yml"
```